pkg/util/mount/mount.go

```diff
@@ -25,7 +25,7 @@ import (
 	"k8s.io/kubernetes/pkg/util/exec"
 )
 
-type Interface interface {
+type Mounter interface {
```

Comment:
@thockin Yeah, agree that we probably don't need this name change.  I actually have suggested this name change myself before.  It is idiomatic go to use `Interface` for the primary interface a package provides and descriptive names for secondary interfaces.